### ISSUE TITLE
Server should not exit when gpu_enable set false

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#494]
**Describe the bug**
```
Milvus Release version: v0.6.0, built at 2019-11-22 07:19.48, with MKL library.
You are using Milvus GPU version

Loading configuration from: ../conf/server_config.yaml
Initializing log config from: ../conf/log_config.conf
Config check fail: GPU not supported. Possible reason: gpu_resource_config.enable is set to false.
ERROR: Milvus server fail to load config file
Milvus server exit...

```

**Steps/Code to reproduce behavior**

**Expected behavior**

**Environment details**
0.6.0-GPU-version

**Screenshots**

**Additional context**

**Comment by scsven** [comment]
fixed in #469 , try it again.

**Comment by del-zhenwu** [comment]
close
